src/printer-postcss.js

```diff
@@ -283,16 +283,35 @@ function genericPrint(path, options, print) {
       return path.call(print, "group");
     }
     case "value-comma_group": {
+      const parent = path.getParentNode();
+      let declParent;
+      let i = 0;
+      do {
+        declParent = path.getParentNode(i++);
+      } while (declParent && declParent.type !== "css-decl");
+
+      const isGridValue =
+        parent.type === "value-value" && declParent.prop.startsWith("grid");
```

Comment:
This should be case insensitive: `declParent.prop.toLowerCase().startsWith("grid")`